csv-bulk-signature/src/main.ts

```diff
@@ -162,9 +157,12 @@ async function zipUpFile() {
     zip.file(file, fileContents);
   }
 
-  try {
-    const buffer = await zip.generateAsync({ type: "nodebuffer" });
+  return zip.generateAsync({ type: "nodebuffer" });
+}
 
+export async function zipUpFiles(files: string[]) {
+  try {
+    const buffer = await createZipFile(files);
     await fs.writeFile(ZIP_FILE, buffer);
```

Comment:
now `zipUpFiles` takes in list of files as argument and calls another function to gather the zip buffer, then we just write to the file system.